pkg/datapath/loader/netlink.go

```diff
@@ -111,7 +75,7 @@ func replaceDatapath(ctx context.Context, opts replaceDatapathOptions) (_ func()
 		}
 
 		if err := bpf.RepinMap(bpf.TCGlobalsPath(), key, ms); err != nil {
-			return nil, fmt.Errorf("repinning map %s: %w", key, err)
+			return nil, nil, fmt.Errorf("repinning map %s: %w", key, err)
```

Comment:
I think the `err` below (L83) now doesn't capture the return value anymore. So this subtly breaks the revert mechanism?

Reply:
Sigh, good call